app/src/main/java/com/bogdan/codeforceswatcher/util/Prefs.kt

```diff
@@ -47,13 +45,9 @@ class Prefs constructor(private val context: Context) {
         private const val KEY_ALARM = "key_alarm"
 
         @SuppressLint("StaticFieldLeak")
-        private var prefs: Prefs? = null
 
-        fun get(): Prefs? {
-            if (prefs == null) {
-                prefs = Prefs(CwApp.app)
-            }
-            return prefs
+        fun get(): Prefs {
```

Comment:
I'm not sure where you've found this implementation, but it's not very logical, I think. Here in `get` method, `Prefs` is created every time you call it. It means there is no need to `get()` method in any way.

Previous implementation used `Singleton` pattern, so `Prefs` object has been created only when needed and only once. I prefer this one as it much more effective.